pyvista/core/filters/uniform_grid.py

```diff
@@ -116,3 +116,57 @@ def extract_subset(self, voi, rate=(1, 1, 1), boundary=False, progress_bar=False
         fixed.field_data.update(result.field_data)
         fixed.copy_meta_from(result)
         return fixed
+
+    def image_fft(self, progress_bar=False):
+        """Fast Fourier Transform.
+
+        The input can have real or complex data in any components and data types,
+        but the output is always complex doubles with real values in component0,
+        and imaginary values in component1. The filter is fastest for images
```

Comment:
What does this mean exactly?
> the output is always complex doubles with real values in component0, and imaginary values in component1

It's not clear to me at all, so perhaps we could make this clearer using NumPy language.

Reply:
I think this is straight from the VTK docs https://vtk.org/doc/nightly/html/classvtkImageFFT.html#details

Reply:
> so perhaps we could make this clearer using NumPy language.

I agree, we may want to provide a better explanation

Reply:
Cleaned this up a bit.